MarvelHeroesApp/Controllers/HeroesViewController.swift

```diff
@@ -11,108 +11,113 @@ import PureLayout
+    indicator.startAnimating()
     
-    lazy var indicator: UIActivityIndicatorView = {
-        let indicator = UIActivityIndicatorView(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height))
-        indicator.backgroundColor = UIColor.white
-        indicator.hidesWhenStopped = true
-        indicator.style = .gray
-        return indicator
-    }()
-    
-    lazy var heroesTableView: UITableView = {
-        let tableView = UITableView(frame: .zero, style: .plain)
-        tableView.delegate = self
-        tableView.dataSource = self
-        tableView.translatesAutoresizingMaskIntoConstraints = false
-        tableView.register(HeroTableViewCell.self, forCellReuseIdentifier: self.cellId)
-        return tableView
-    }()
+    addingSubviews()
+    setupConstraints()
+    loadingCharacters()
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        self.view.backgroundColor = .white
-        self.indicator.startAnimating()
-
-        self.addSubviews()
-        self.setupConstraints()
-        self.loadCharacters()
-        
-        if realm.objects(Hero.self).count != 0 {
-            favoriteHero = realm.objects(Hero.self)[0].nameHero
-        }
+    if realm.objects(Hero.self).count != 0 {
+      favoriteHero = realm.objects(Hero.self)[0].nameHero
     }
-    
-    func addSubviews() {
-        self.view.addSubview(heroesTableView)
-        self.view.addSubview(indicator)
-
-    }
-    
-    func setupConstraints() {
-        heroesTableView.autoPinEdge(toSuperviewEdge: .top, withInset: UIApplication.shared.statusBarFrame.height)
-        heroesTableView.autoPinEdge(toSuperviewEdge: .bottom, withInset: self.tabBarController!.tabBar.frame.height)
-        heroesTableView.autoPinEdge(toSuperviewEdge: .right, withInset: 0)
-        heroesTableView.autoPinEdge(toSuperviewEdge: .left, withInset: 0)
-    }
-    
-    func loadCharacters() {
-        self.requestModel.getCharacters(page: self.currentPage, success: { [weak self] (success) in
-            self?.heroes += success
-            self?.heroesTableView.reloadData()
-            UIView.animate(withDuration: 0.5, animations: {
-                self?.indicator.alpha = 0
-            }, completion: { (bool) in
-                self?.indicator.stopAnimating()
-            })
-        }) { (failure) in
-            self.indicator.stopAnimating()
-        }
+  }
+  
+  private func addingSubviews() {
+    view.addSubview(heroesTableView)
+    view.addSubview(indicator)
+  }
+  
+  private func setupConstraints() {
+    heroesTableView.autoPinEdge(toSuperviewEdge: .top, withInset: UIApplication.shared.statusBarFrame.height)
+    heroesTableView.autoPinEdge(toSuperviewEdge: .bottom, withInset: tabBarController!.tabBar.frame.height)
+    heroesTableView.autoPinEdge(toSuperviewEdge: .right, withInset: 0)
+    heroesTableView.autoPinEdge(toSuperviewEdge: .left, withInset: 0)
+  }
+  
+  private func loadingCharacters() {
```

Comment:
Let's use -ing suffixes only for non-mutating functions (methods that don't modify self)